src/coreclr/src/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -585,7 +585,7 @@ internal static unsafe IntPtr ConvertToNative(string managed)
             if (!Environment.IsWinRTSupported)
                 throw new PlatformNotSupportedException(SR.PlatformNotSupported_WinRT);
             if (managed == null)
-                throw new ArgumentNullException(); // We don't have enough information to get the argument name
+                throw new ArgumentNullException(nameof(managed)); // We don't have enough information to get the argument name
```

Comment:
Agreed. This will never make sense to the caller since this is called during an interop scenario that the user largely has no control over.

Reply:
> The change and the existing comment are at odds. Since this is an internal method I assume the comment means that we no longer know what the name of the parameter the input method used was... which seems fair.

OK
> So I'd undo both changes in this file, and change them to supressions.

I did not suppressed any warning which I think false positive, because the warning is info level so not failing the build. Should I suppress them?

Reply:
I would have targeted suppression here. The issue is even as info I believe they pollute the build logs which is something that is already noisy. If I am wrong and there are no messages about these locations during build then I am fine leaving as is.

Reply:
I didn't see info level warnings logged, but will double check